### DISCUSSION TITLE
[Question][Hw4] Gradescope autograder does not finish

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs424-program-reasoning#342] [Q&A]
I made a submission at 1:27pm with ID 147442588. However, the autograder does not seem to work, and I still have the "The autograder hasn't finished running yet." message after an hour. Is there a problem with the grader?

Thanks in advance

**Comment by sujin0529** [comment]
Hi,

Sorry, I don't know why it took more than an hour to grade.
But, now that I checked, the autograder seems to be working well.
If the same problem occurs again, please resubmit it.

Thanks.

**Comment by Lee-Janggun** [comment]
I made a resubmission, and it worked fine. Thanks for the check!
